util/db_options.cc

```diff
@@ -0,0 +1,240 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
```

Comment:
If I remember correctly, cc_options.cc should have already been moved to options/.